exonum-java-binding/core/src/main/java/com/exonum/binding/core/runtime/ServiceWrapper.java

```diff
@@ -80,7 +105,24 @@ void initialize(Fork view, Configuration configuration) {
     service.initialize(view, configuration);
   }
 
-  void executeTransaction(int txId, byte[] arguments, TransactionContext context)
+  void executeTransaction(String interfaceName, int txId, byte[] arguments, int callerServiceId,
+      TransactionContext context)
+      throws TransactionExecutionException {
+    switch (interfaceName) {
+      case DEFAULT_INTERFACE_NAME: {
+        executeIntrinsicTransaction(txId, arguments, context);
+        break;
+      }
+      case CONFIGURE_INTERFACE_NAME: {
+        executeConfigurableTransaction(txId, arguments, callerServiceId, context);
+        break;
+      }
+      default: throw new IllegalArgumentException(
+          format("Unknown interface (name=%s, txId=%d)", interfaceName, txId));
```

Comment:
Why static import for `String.format`? That's inconsistent with the rest of our codebase

Reply:
To have it more consice and fit in a single expression. I wouldn't consider the consistency of static imports of a particular method as something to be given higher priority over readability and local consistency.